### DISCUSSION TITLE
running tests: yarn not installed causes test error

### DISCUSSION THREAD

**Discussion by dmvjs** [vercel/next.js#37414] [Help] [answered]
in `.github / actions / next-stats-action/src/prepare/repo-setup.js` : 130 a string likely eval-ed includes a yarn invocation but installing yarn wasn't mentioned in contributing guide explicitly when using pnpm. globally installing yarn fixed the issue.

<img width="1276" alt="image" src="https://user-images.githubusercontent.com/1675992/171707445-a7874d52-aa09-4077-a718-6bcf19abbc8f.png">

<img width="855" alt="image" src="https://user-images.githubusercontent.com/1675992/171707651-d82884a1-36eb-49e3-8264-c3089c27aa7e.png">

This section could include a call to install yarn globally, or the issue could be remedied by using pnpm or whatever *pm du jour in the eval code above 👆

![image](https://user-images.githubusercontent.com/1675992/171729420-27e91c58-119b-48d1-a58a-24779ac6a561.png)

**Comment by icyJoseph** [comment, marked as answer]
Yeah I think this is all coming in over the past 10 days, if not less.  https://github.com/vercel/next.js/pull/37259, there'll be many places where things ought to be updated.
